spec/services/attribute_asserter_spec.rb

```diff
@@ -25,106 +24,22 @@
       metadata: {},
     )
   end
-  let(:raw_sp1_authn_request) do
-    sp1_authnrequest = saml_authn_request_url(overrides: { issuer: sp1_issuer })
-    CGI.unescape sp1_authnrequest.split('SAMLRequest').last
-  end
-  let(:raw_aal3_sp1_authn_request) do
-    ial1_aal3_authnrequest = saml_authn_request_url(
-      overrides: {
-        issuer: sp1_issuer,
-        authn_context: [
-          Saml::Idp::Constants::IAL1_AUTHN_CONTEXT_CLASSREF,
-          Saml::Idp::Constants::AAL3_AUTHN_CONTEXT_CLASSREF,
-        ],
-      },
-    )
-    CGI.unescape ial1_aal3_authnrequest.split('SAMLRequest').last
-  end
-  let(:raw_ial1_authn_request) do
-    ial1_authn_request_url = saml_authn_request_url(
-      overrides: {
-        issuer: sp1_issuer,
-        authn_context: Saml::Idp::Constants::IAL1_AUTHN_CONTEXT_CLASSREF,
-      },
-    )
-    CGI.unescape ial1_authn_request_url.split('SAMLRequest').last
-  end
-  let(:raw_vtr_no_proofing_authn_request) do
-    vtr_proofing_authn_request = saml_authn_request_url(
-      overrides: {
-        issuer: sp1_issuer,
-        authn_context: 'C1.C2',
-      },
-    )
-    CGI.unescape vtr_proofing_authn_request.split('SAMLRequest').last
-  end
-  let(:raw_ial2_authn_request) do
-    ial2_authnrequest = saml_authn_request_url(
-      overrides: {
-        issuer: sp1_issuer,
-        authn_context: Saml::Idp::Constants::IAL2_AUTHN_CONTEXT_CLASSREF,
-      },
-    )
-    CGI.unescape ial2_authnrequest.split('SAMLRequest').last
-  end
-  let(:raw_vtr_proofing_authn_request) do
-    vtr_proofing_authn_request = saml_authn_request_url(
-      overrides: {
-        issuer: sp1_issuer,
-        authn_context: 'C1.C2.P1',
-      },
-    )
-    CGI.unescape vtr_proofing_authn_request.split('SAMLRequest').last
-  end
-  let(:raw_ial1_aal3_authn_request) do
-    ial1_aal3_authnrequest = saml_authn_request_url(
-      overrides: {
-        issuer: sp1_issuer,
-        authn_context: [
-          Saml::Idp::Constants::IAL1_AUTHN_CONTEXT_CLASSREF,
-          Saml::Idp::Constants::AAL3_AUTHN_CONTEXT_CLASSREF,
-        ],
-      },
-    )
-    CGI.unescape ial1_aal3_authnrequest.split('SAMLRequest').last
-  end
-  let(:raw_ialmax_authn_request) do
-    ialmax_authnrequest = saml_authn_request_url(
+
+  let(:options) { {} }
+  let(:raw_authn_request) do
+    raw = saml_authn_request_url(
       overrides: {
         issuer: sp1_issuer,
```

Comment:
i'm wondering why we are using this sp1_issuer, rather than the issuer of the service_provider created here, but i haven't looked into that yet